<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSetPosition</class>
 <widget class="QDialog" name="CSetPosition">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set map position</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_1">
        <attribute name="title">
         <string>R.A./Dec</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Position</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_6">
              <item row="3" column="0">
               <widget class="QSpinBox" name="spinBox_5">
                <property name="minimumSize">
                 <size>
                  <width>50</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="minimum">
                 <number>-90</number>
                </property>
                <property name="maximum">
                 <number>90</number>
                </property>
               </widget>
              </item>
              <item row="1" column="4">
               <widget class="QSpinBox" name="spinBox_3">
                <property name="minimumSize">
                 <size>
                  <width>50</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximum">
                 <number>59</number>
                </property>
               </widget>
              </item>
              <item row="3" column="2">
               <widget class="QSpinBox" name="spinBox_4">
                <property name="minimumSize">
                 <size>
                  <width>50</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximum">
                 <number>59</number>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Â°</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QSpinBox" name="spinBox_2">
                <property name="minimumSize">
                 <size>
                  <width>50</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximum">
                 <number>59</number>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>h</string>
                </property>
               </widget>
              </item>
              <item row="1" column="5">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>s</string>
                </property>
               </widget>
              </item>
              <item row="3" column="5">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>&quot;</string>
                </property>
               </widget>
              </item>
              <item row="3" column="3">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>'</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QSpinBox" name="spinBox">
                <property name="minimumSize">
                 <size>
                  <width>50</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximum">
                 <number>23</number>
                </property>
               </widget>
              </item>
              <item row="2" column="0" colspan="3">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Declination</string>
                </property>
               </widget>
              </item>
              <item row="1" column="3">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>m</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0" colspan="3">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>R.A.</string>
                </property>
               </widget>
              </item>
              <item row="3" column="4">
               <widget class="QSpinBox" name="spinBox_6">
                <property name="minimumSize">
                 <size>
                  <width>50</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximum">
                 <number>59</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Alt/Azm</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Position</string>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0" colspan="3">
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>Azimuth</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QSpinBox" name="spinBox_7">
                <property name="minimumSize">
                 <size>
                  <width>50</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximum">
                 <number>359</number>
                </property>
               </widget>
              </item>
              <item row="3" column="2">
               <widget class="QSpinBox" name="spinBox_10">
                <property name="minimumSize">
                 <size>
                  <width>50</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximum">
                 <number>59</number>
                </property>
               </widget>
              </item>
              <item row="1" column="5">
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>&quot;</string>
                </property>
               </widget>
              </item>
              <item row="3" column="3">
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>'</string>
                </property>
               </widget>
              </item>
              <item row="1" column="4">
               <widget class="QSpinBox" name="spinBox_9">
                <property name="minimumSize">
                 <size>
                  <width>50</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximum">
                 <number>59</number>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QSpinBox" name="spinBox_11">
                <property name="minimumSize">
                 <size>
                  <width>50</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="minimum">
                 <number>-90</number>
                </property>
                <property name="maximum">
                 <number>90</number>
                </property>
               </widget>
              </item>
              <item row="3" column="4">
               <widget class="QSpinBox" name="spinBox_12">
                <property name="minimumSize">
                 <size>
                  <width>50</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximum">
                 <number>59</number>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLabel" name="label_17">
                <property name="text">
                 <string>Â°</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QSpinBox" name="spinBox_8">
                <property name="minimumSize">
                 <size>
                  <width>50</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximum">
                 <number>59</number>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>Â°</string>
                </property>
               </widget>
              </item>
              <item row="1" column="3">
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>'</string>
                </property>
               </widget>
              </item>
              <item row="3" column="5">
               <widget class="QLabel" name="label_15">
                <property name="text">
                 <string>&quot;</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0" colspan="3">
               <widget class="QLabel" name="label_16">
                <property name="text">
                 <string>Altitude</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Ecliptic</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string>Position</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_9">
              <item row="0" column="0" colspan="3">
               <widget class="QLabel" name="label_18">
                <property name="text">
                 <string>Longitude</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QSpinBox" name="spinBox_13">
                <property name="minimumSize">
                 <size>
                  <width>50</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximum">
                 <number>359</number>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="label_19">
                <property name="text">
                 <string>Â°</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QSpinBox" name="spinBox_14">
                <property name="minimumSize">
                 <size>
                  <width>50</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximum">
                 <number>59</number>
                </property>
               </widget>
              </item>
              <item row="1" column="3">
               <widget class="QLabel" name="label_20">
                <property name="text">
                 <string>'</string>
                </property>
               </widget>
              </item>
              <item row="1" column="4">
               <widget class="QSpinBox" name="spinBox_15">
                <property name="minimumSize">
                 <size>
                  <width>50</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximum">
                 <number>59</number>
                </property>
               </widget>
              </item>
              <item row="1" column="5">
               <widget class="QLabel" name="label_21">
                <property name="text">
                 <string>&quot;</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0" colspan="3">
               <widget class="QLabel" name="label_24">
                <property name="text">
                 <string>Latitude</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QSpinBox" name="spinBox_17">
                <property name="minimumSize">
                 <size>
                  <width>50</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="minimum">
                 <number>-90</number>
                </property>
                <property name="maximum">
                 <number>90</number>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLabel" name="label_25">
                <property name="text">
                 <string>Â°</string>
                </property>
               </widget>
              </item>
              <item row="3" column="2">
               <widget class="QSpinBox" name="spinBox_16">
                <property name="minimumSize">
                 <size>
                  <width>50</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximum">
                 <number>59</number>
                </property>
               </widget>
              </item>
              <item row="3" column="3">
               <widget class="QLabel" name="label_22">
                <property name="text">
                 <string>'</string>
                </property>
               </widget>
              </item>
              <item row="3" column="4">
               <widget class="QSpinBox" name="spinBox_18">
                <property name="minimumSize">
                 <size>
                  <width>50</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximum">
                 <number>59</number>
                </property>
               </widget>
              </item>
              <item row="3" column="5">
               <widget class="QLabel" name="label_23">
                <property name="text">
                 <string>&quot;</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>Keep FOV</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider">
         <property name="maximum">
          <number>90</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_26">
         <property name="text">
          <string>Rotation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox">
         <property name="suffix">
          <string>Â°</string>
         </property>
         <property name="minimum">
          <double>-90.000000000000000</double>
         </property>
         <property name="maximum">
          <double>90.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>OK</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>spinBox</tabstop>
  <tabstop>spinBox_2</tabstop>
  <tabstop>spinBox_3</tabstop>
  <tabstop>spinBox_5</tabstop>
  <tabstop>spinBox_4</tabstop>
  <tabstop>spinBox_6</tabstop>
  <tabstop>spinBox_7</tabstop>
  <tabstop>spinBox_8</tabstop>
  <tabstop>spinBox_9</tabstop>
  <tabstop>spinBox_11</tabstop>
  <tabstop>spinBox_10</tabstop>
  <tabstop>spinBox_12</tabstop>
  <tabstop>spinBox_13</tabstop>
  <tabstop>spinBox_14</tabstop>
  <tabstop>spinBox_15</tabstop>
  <tabstop>spinBox_17</tabstop>
  <tabstop>spinBox_16</tabstop>
  <tabstop>spinBox_18</tabstop>
  <tabstop>horizontalSlider</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>spinBox_5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>CSetPosition</receiver>
   <slot>check()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>CSetPosition</receiver>
   <slot>check()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>376</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_6</sender>
   <signal>valueChanged(int)</signal>
   <receiver>CSetPosition</receiver>
   <slot>check()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_11</sender>
   <signal>valueChanged(int)</signal>
   <receiver>CSetPosition</receiver>
   <slot>check()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_10</sender>
   <signal>valueChanged(int)</signal>
   <receiver>CSetPosition</receiver>
   <slot>check()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_12</sender>
   <signal>valueChanged(int)</signal>
   <receiver>CSetPosition</receiver>
   <slot>check()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>check()</slot>
 </slots>
</ui>
